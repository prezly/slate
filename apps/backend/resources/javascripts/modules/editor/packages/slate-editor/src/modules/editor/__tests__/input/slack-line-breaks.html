<meta charset='utf-8'><span style="color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">¿Sabían que en 30 años se duplicará la población con más de 65 años en América Latina?</span><span class="c-mrkdwn__br" data-stringify-type="paragraph-break" style="box-sizing: inherit; display: block; height: 8px; color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial;"></span><span style="color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">En este contexto, los invitamos a sumarse a la presentación del Reporte de Desarrollo y Economía de<span> </span></span><span class="c-mrkdwn__highlight" style="box-sizing: inherit; background: rgba(232, 145, 45, 0.4); color: rgba(var(--sk_secondary_highlight,242,199,68),1); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-style: initial; text-decoration-color: initial;">CAF</span><span style="color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"><span> </span>- Banco de desarrollo de América Latina, donde expertos en materia fiscal hablarán sobre el envejecimiento, la informalidad y los cambios tecnológicos como desafíos para mejorar la cobertura, calidad y sostenibilidad de la protección social en América Latina.</span><span class="c-mrkdwn__br" data-stringify-type="paragraph-break" style="box-sizing: inherit; display: block; height: 8px; color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial;"></span><span style="color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">¡Comienza en breve!</span><span class="c-mrkdwn__br" data-stringify-type="paragraph-break" style="box-sizing: inherit; display: block; height: 8px; color: rgb(209, 210, 211); font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41); text-decoration-style: initial; text-decoration-color: initial;"></span><a target="_blank" class="c-link" data-stringify-link="https://www.youtube.com/watch?v=Y2Hez3QWc_I&amp;feature=youtu.be" delay="150" data-sk="tooltip_parent" href="https://www.youtube.com/watch?v=Y2Hez3QWc_I&amp;feature=youtu.be" rel="noopener noreferrer" style="box-sizing: inherit; color: rgba(var(--sk_highlight,18,100,163),1); text-decoration: none; font-family: Slack-Lato, appleLogo, sans-serif; font-size: 15px; font-style: normal; font-variant-ligatures: common-ligatures; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 37, 41);">https://www.youtube.com/watch?v=Y2Hez3QWc_I&amp;feature=youtu.be</a>
