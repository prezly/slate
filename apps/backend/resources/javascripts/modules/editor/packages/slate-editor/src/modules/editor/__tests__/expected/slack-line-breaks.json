[
    {
        "type": "paragraph",
        "children": [
            {
                "text": "¿Sabían que en 30 años se duplicará la población con más de 65 años en América Latina?"
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": "En este contexto, los invitamos a sumarse a la presentación del Reporte de Desarrollo y Economía de CAF - Banco de desarrollo de América Latina, donde expertos en materia fiscal hablarán sobre el envejecimiento, la informalidad y los cambios tecnológicos como desafíos para mejorar la cobertura, calidad y sostenibilidad de la protección social en América Latina."
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": "¡Comienza en breve!"
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": ""
            },
            {
                "href": "https://www.youtube.com/watch?v=Y2Hez3QWc_I&feature=youtu.be",
                "type": "link",
                "children": [
                    {
                        "text": "https://www.youtube.com/watch?v=Y2Hez3QWc_I&feature=youtu.be"
                    }
                ]
            },
            {
                "text": ""
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": ""
            }
        ]
    }
]
